src/test/Drips.t.sol

```diff
@@ -65,28 +75,53 @@ contract DripsTest is DSTest {
         }
     }
 
-    function dripsReceivers() internal pure returns (DripsReceiver[] memory list) {
+    function recv() internal pure returns (DripsReceiver[] memory list) {
         list = new DripsReceiver[](0);
     }
 
-    function dripsReceivers(uint256 userId, uint128 amtPerSec)
+    function recv(uint256 userId, uint256 amtPerSec)
         internal
         pure
-        returns (DripsReceiver[] memory list)
+        returns (DripsReceiver[] memory receivers)
     {
-        list = new DripsReceiver[](1);
-        list[0] = DripsReceiver(userId, amtPerSec, 0, 0);
+        return recv(userId, amtPerSec, 0, 0);
     }
 
-    function dripsReceivers(
-        uint256 userId1,
-        uint128 amtPerSec1,
-        uint256 userId2,
-        uint128 amtPerSec2
-    ) internal pure returns (DripsReceiver[] memory list) {
-        list = new DripsReceiver[](2);
-        list[0] = DripsReceiver(userId1, amtPerSec1, 0, 0);
-        list[1] = DripsReceiver(userId2, amtPerSec2, 0, 0);
+    function recv(
+        uint256 userId,
+        uint256 amtPerSec,
+        uint256 start,
+        uint256 duration
+    ) internal pure returns (DripsReceiver[] memory receivers) {
+        receivers = new DripsReceiver[](1);
+        receivers[0] = DripsReceiver(userId, uint128(amtPerSec), uint64(start), uint64(duration));
+    }
+
+    function recv(DripsReceiver[] memory recv1, DripsReceiver[] memory recv2)
```

Comment:
nice helpers

Reply:
One idea to make it really nice could be to add the params in a struct.


Then you could call it with:

```solidity
recv(Receiver({amtPerSec: 5, ....}))
```

Which would allow to easily see what each parameter means :)

Reply:
You don't need to use the `Receiver` struct directly, in Solidity you can just name the parameters of the helper if you need that extra clarity:
```
recv({userId: receiver, amtPerSec: 5, start: 0, duration: 10})
```